src/components/SpecialResolution/SpecialResolutionForm.vue

```diff
@@ -368,6 +372,7 @@ export default class SpecialResolutionForm extends Mixins(DateMixin) {
       ...this.getcreateResolution,
       signingDate: val
     })
+    this.validate()
```

Comment:
I'm not sure why we need to do validation here.

Reply:
We need this to set the component state valid. so when there is change component state will set as valid or not

Reply:
Alright.